decidim-accountability/spec/commands/admin/import_projects_to_accountability_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim::Accountability
+  describe Admin::ImportProjectsToAccountability do
+    # subject { described_class.new(form) }
+
+    let(:user) { create :user, organization: organization }
+
+    let(:organization) { create(:organization) }
+    let(:participatory_space) { create(:participatory_process, organization: organization) }
+
+    let(:current_component) { create(:component, manifest_name: "accountability", participatory_space: participatory_space, published_at: accountability_component_published_at) }
+    let(:accountability_component_published_at) { nil }
+
+    # let(:scope) { create :scope, organization: organization }
+    # let(:category) { create :category, participatory_space: participatory_space }
+    # let(:start_date) { Date.yesterday }
+    # let(:end_date) { Date.tomorrow }
+
+    let(:budget_component) { create(:component, manifest_name: "budgets", participatory_space: participatory_space) }
+    let(:budget) { create(:budget, component: budget_component, total_budget: 26_000_000) }
+    let!(:project) { create(:project, budget: budget, selected_at: selected_at) }
+    let(:selected_at) { Time.current }
+    let(:weight) { 0.3 }
+    let(:external_id) { "external-id" }
+    let(:progress) { 89 }
+
+    let(:command) { described_class.new(form) }
+    let(:proposal_component) do
+      create(:component, manifest_name: "proposals", participatory_space: participatory_space)
+    end
+
+    let(:project_component) do
```

Comment:
Don't leave any memoized helpers you are not using. Quickly searching through I don't see this used anywhere (?).